docs/4.1. Behaviour - Authorization Servers.md

```diff
@@ -67,27 +67,26 @@ all OAuth 2.0 registration mechanisms they support.
 
 ### Client Authentication
 
-Authorization Servers MUST provide support for client authentication at the token and revocation endpoints using HTTP
-Basic Authentication as per Section 2 of [RFC 2617][RFC-2617], and in the manner described in Section 2.3 of
+Authorization Servers MUST provide support client authentication at the token and revocation endpoints using the
+following methods. Additional methods MAY also be supported.
+*   HTTP Basic Authentication as per Section 2 of [RFC 2617][RFC-2617], and in the manner described in Section 2.3 of
 [RFC 6749][RFC-6749]. This mechanism is identified by the client registration `token_endpoint_auth_method` of
 `client_secret_basic`.
-
-Authorization Servers MUST provide support for client authentication at the token and revocation endpoints using a
-digitally signed JSON Web Token in the manner described in Section 2.2 of [RFC 7523][RFC-7523]. This mechanism is
+*   Digitally signed JSON Web Tokens in the manner described in Section 2.2 of [RFC 7523][RFC-7523]. This mechanism is
 identified by the client registration `token_endpoint_auth_method` of `private_key_jwt`.
 
 The security considerations outlined in Section 10.1 of [RFC 6749][RFC-6749] SHOULD be abided by with regards to client
 authentication.
 
 ### Client De-Registration
 
-Authorization Servers SHOULD support the de-registration of clients in the case of a compromised client. The means
-by which this client de-registration occurs is beyond the scope of this specification.
+Authorization Servers SHOULD support de-registration of clients in the case of a compromised client. The means by which
+this client de-registration occurs is beyond the scope of this specification.
```

Comment:
_de-registration_ does not take a hyphen according to google

```suggestion
Authorization Servers SHOULD support unregistering clients in the case of a compromised client. The means by
which the client is unregistered is beyond the scope of this specification.
```

Reply:
I agree that deregistration doesn't need a hyphen, but I don't agree that using unregistered is the right fix.
https://english.stackexchange.com/questions/25931/unregister-vs-deregister

Reply:
I read that as well 🤣 I very mildly disagree, perhaps it's cultural but my tendency leans on `un-
https://english.stackexchange.com/a/25933

```suggestion
Authorization Servers SHOULD support deregistering clients in the case of a compromised client. The means by
which the client is deregistered is beyond the scope of this specification.
```

Reply:
I've removed the hyphen for now, but happy to adjust further if desired.